### DISCUSSION TITLE
What Software supports opentelemetry.proto.collector.profiles.v1experimental.ProfilesService?

### DISCUSSION THREAD

**Discussion by RononDex** [open-telemetry/opentelemetry-collector-contrib#36449] [Q&A]
I am currently trying to test life profiling of processes using the [Open Telemetry eBPF profiler](https://github.com/open-telemetry/opentelemetry-ebpf-profiler). After enabling the feature gate `service.profilesSupport` the eBPF profiler can send data to my Open Telemetry Collector (Contrib). However, the Collector seems unable to export it to jaeger with the following error message:

```
024-11-19T13:01:49.975Z        error   internal/queue_sender.go:92     Exporting failed. Dropping data.        {"kind": "exporter", "data_type": "profiles", "name": "otlp", "error": "not retryable error: Permanent error: rpc error: code = Unimplem
ented desc = unknown service opentelemetry.proto.collector.profiles.v1experimental.ProfilesService", "dropped_items": 32}
go.opentelemetry.io/collector/exporter/exporterhelper/internal.NewQueueSender.func1
        go.opentelemetry.io/collector/exporter@v0.113.0/exporterhelper/internal/queue_sender.go:92
go.opentelemetry.io/collector/exporter/internal/queue.(*Consumers[...]).Start.func1
        go.opentelemetry.io/collector/exporter@v0.113.0/internal/queue/consumers.go:43

```

Hinting that the current jeager version does not support Profiling information. What Software could I use to export the Profiling data to? Is there any Software available capable of handling profiling data yet?

**Comment by dalbani** [comment]
The presentation at https://static.sched.com/hosted_files/kccncna2024/46/Continuous%20Profiling%20OTel%20-%20Christos%20Kalkanis.pdf from 2024 mentions the following backends supporting profile signals:
> Profiling agent can talk to any service that supports OTLP Profiling signal: Elastic, Polar Signals, Pyroscope, Datadog (coming soon)
